src/inboxsdk-js/loading/load-platform-implementation-NONREMOTE.ts

```diff
@@ -1,17 +1,9 @@
 import once from 'lodash/once';
 
-export default function (delay: number): () => Promise<void> {
```

Comment:
The rub here is whether or not the delay way load bearing. With

```
  await new Promise(resolve => setTimeout(resolve, delay));
```

I couldn't get an example extension to start up.

Reply:
Throwing this in and smoothing things should be something we do as part of this.